cfy_manager/main.py

```diff
@@ -146,11 +150,28 @@ def _print_finish_message():
     logger.notice('#' * 50)
 
 
+def _is_db_initiated():
+    return os.path.isfile(INITIAL_DB_CREATION_FILE)
+
+
+def _update_initial_db_file():
```

Comment:
maybe create the file when the db is created, not at the end of the whole process

Reply:
Maybe it was a poor choice of file name. But the idea was - I only create it after a full successful install. On the assumption that the users will keep re-running install until it succeeds at least once.

Reply:
oh right, it's actually a good idea to keep rebuilding the db after failed runs